assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -94,8 +94,10 @@ che.workspace.terminal_linux_arm7=${che.home}/lib/linux_arm7/terminal
 
 che.workspace.exec_linux_amd64=${che.home}/lib/linux_amd64/exec
 
-# RAM default for each machine that has no RAM settings in environment.
-che.workspace.default_memory_mb=1024
+# RAM limit default for each machine that has no RAM settings in environment.
+che.workspace.default_memory_limit_mb=2048
```

Comment:
you have to use aliases for backward compatibility with ```che.workspace.default_memory_mb``` property.  BTW why default value is 2G?

Reply:
sorry i pushed commit to the wrong branch, and overwritten this pr by mistake. i will fix this shortly, and create a new PR for this